Makefile

```diff
@@ -19,6 +19,9 @@ help:
 clean:
 	@$(SPHINXBUILD) -M clean "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 	# remove auto-generated content
+	rm -rf src/plugins/main-loop/built-in
+	rm -rf src/user-guide/task-implementation/job-runner-handlers
+	# remove auto-generated content (old)
 	rm -rf src/user-guide/plugins/main-loop/built-in
 	rm -rf src/user-guide/batch-sys-handlers
 	rm -rf src/user-guide/job-runner-handlers
```

Comment:
```suggestion
```

Reply:
The idea was to remove old files that might be kicking around due to checking out between different branches, is that not needed?

Reply:
Can do with `git clean` rather than leaving old paths in a makefile.